tests/unit/tidy_output_test_files/tidy6.out

```diff
@@ -0,0 +1,9 @@
+files/test5.cpp:10:9: error: no matching function for call to 'get_type' [clang-diagnostic-error]
+  (void)get_type(ptr); // expected-error{{no matching function for call to 'get_type'}}
```

Comment:
Could we change these test related comments to something different? Remove it or add other non test infrastructure related comments.